rotkehlchen/chain/evm/node_inquirer.py

```diff
@@ -214,6 +214,7 @@ def __init__(
         self.database = database
         self.blockchain = blockchain
         self.etherscan = etherscan
+        self.etherscan_block: BlockNumber = BlockNumber(0)  # caches highest block for checking sync while connecting to nodes  # noqa: E501
```

Comment:
nitpick: Doesn't it understand the type from the assignment?

```suggestion
        self.etherscan_block = BlockNumber(0)  # caches highest block for checking sync while connecting to nodes  # noqa: E501
```